app/models/user.rb

```diff
@@ -1,9 +1,12 @@
 class User < ApplicationRecord
-  has_many :posts
-  has_many :likes
+  has_many :posts, foreign_key: 'user_id'
   has_many :comments
+  has_many :likes, foreign_key: 'user_id'
+
+  validates :name, presence: true
+  validates :posts_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validates :bio, presence: true, length: { minimum: 20, maximum: 180 }
 
-  # method that returns the 3 most recent posts for a given user.
   def recent_posts
     posts.order(created_at: :desc).limit(3)
   end
```

Comment:
[OPTIONAL] Because the previous reviewer didn't notice this the following suggestion is NOT mandatory. I do recommend strongly for you to have a look at it.
- [ ] Any custom methods in the models (User, Post, Like, and Comment) should have a corresponding unit test. (Ex: recent_posts method should have a unit test).